src/tests/IntegrationTests/Events/RepositoryManagerTests.cs

```diff
@@ -435,6 +469,7 @@ await RepositoryManager
 
                 repositoryManagerListener.Received().OnIsBusyChanged(Args.Bool);
                 repositoryManagerListener.Received().CurrentBranchUpdated(Args.NullableConfigBranch, Args.NullableConfigRemote);
+                repositoryManagerListener.DidNotReceive().GitAheadBehindStatusUpdated(Args.GitAheadBehindStatus);
```

Comment:
Removing a remote might affect the remote for the current branch. We should be triggering this here.